prereise/gather/winddata/hrrr/hrrr.py

```diff
@@ -0,0 +1,19 @@
+from prereise.gather.winddata.hrrr.constants import HRRR_S3_BASE_URL
+from prereise.gather.winddata.hrrr.downloader import Downloader
+from prereise.gather.winddata.hrrr.hrrr_api import HrrrApi
+
+
+def retrieve_data(start_dt, end_dt, directory):
+    """Retrieves all HRRR wind data for all hours between start_dt and
+    end_dt. (In a future PR) will convert all that wind data to
+    Pout in order to be compatible with REISE.
+
+    Sample usage:
```

Comment:
Nice, worked for me. Should we include the `datetime` and `timedelta` imports in the sample?